partiql-eval/src/eval.rs

```diff
@@ -199,14 +210,9 @@ impl Evaluable for EvalScan {
     }
 }
 
-#[derive(Debug)]
-pub enum EvalJoinKind {
-    Inner,
-    Left,
-    Right,
-    Full,
-}
-
+/// Represents an evaluation `Join` operator; `Join` joins the tuples from its LHS and RHS based on a logic defined
+/// by [`EvalJoinKind`]. For semantics of PartiQL joins and their distinction with SQL's see sections
+/// 5.3 – 5.7 of [PartiQL Specification — 2007](https://partiql.org/assets/PartiQL-Specification.pdf).
```

Comment:
What does the "2007" mean?

Reply:
Was supposed to be the spec year but not sure where `2007` has come from. The idea is to cater for future spec versions.
Fixed in the new commit.